src/timesync/timesyncd-manager.c

```diff
@@ -373,7 +373,7 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
                 return -errno;
 
         /* If touch fails, there isn't much we can do. Maybe it'll work next time. */
-        (void) touch("/var/lib/systemd/clock");
+        (void) touch("/var/lib/systemd/timesync/clock");
 
```

Comment:
there's one more mention of /var/lib/systemd/clock left: in the man page `man/systemd-timesyncd.service.xml`. That needs to be patched too.

Reply:
Oh, thanks.